02-introducing-survey-data.Rmd

```diff
@@ -1,2 +1,188 @@
+
+Table: Protocol Example for 6-month Web and CAPI Data Collection
+
+Week  | Contact Mode          | Contact Message         | Survey Mode Offered
+:----: | ------------------------------ | ------------------------------- | -------------------
+   1 | Mail: Letter          | Prenotice            | ---
+   2 | Mail: Letter          | Invitation           | Web
+   3 | Mail: Postcard         | Thank You/Reminder       | Web
+   6 | Mail: Letter in large envelope | Animal Welfare Discussion    | Web
+  10 | Mail: Postcard         | Inform Upcoming In-Person Visit | Web
+  14 | In-Person Visit        | ---               | CAPI
+  16 | Mail: Letter          | Reminder of In-Person Visit   | Web, but includes number to call to schedule CAPI
+  20 | In-Person Visit        | ---               | CAPI
+  25 | Mail: Letter in large envelope | Survey Closing Notice      | Web, but includes number to call to schedule CAPI
+
+This is just one possible protocol that could be used that starts respondents with web (typically done to reduce costs). However, researchers may want to start in-person data collection earlier during the data collection period, or ask their interviewers to attempt more than 2 visits with a household.
+
+### Questionnaire Design {#overview-design-questionnaire}
+
+When developing the questionnaire, it can be helpful to first outline the topics to be asked and include the "why" each question or topic is important to the research question(s). This can help researchers better tailor the questionnaire and potentially reduce the number of questions (and thus burden on the respondent) if topics are deemed irrelevant to the research question. When making these decisions, researchers should also consider questions needed for weighting purposes. While we would love to have everyone sampled answer our survey, this is rarely the case. Thus, including questions about demographics in the survey can assist with weighting for *nonresponse error* (both unit and item nonresponse). Knowing details of the sampling plan and what may impact *coverage error* and *sampling error* can help guide researchers in determining what types of demographics to include.
+
+Researchers can benefit from the work of others by using questions from other surveys. This is common with demographic questions such as race and ethnicity that use questions from a government census or other official surveys. Other survey questions can be found using question banks which are a compilation of questions that have been asked across a variety of surveys such as the [Inter-university Consortium for Political and Social Research (ICPSR) variable search] (https://www.icpsr.umich.edu/web/pages/ICPSR/ssvd/).
```

Comment:
I've moved URL to footnote so it works in both printed and web versions. Let's discuss in meeting on how we want to link to external URLs. Footnotes or something else?

```suggestion
Researchers can benefit from the work of others by using questions from other surveys. This is common with demographic questions such as race and ethnicity that use questions from a government census or other official surveys. Other survey questions can be found using question banks which are a compilation of questions that have been asked across a variety of surveys such as the Inter-university Consortium for Political and Social Research (ICPSR) variable search^[See: https://www.icpsr.umich.edu/web/pages/ICPSR/ssvd/].
```